sh/core.inc.sh

```diff
@@ -28,9 +53,14 @@ GIT_REMOTE=$(git config branch.${GIT_BRANCH}.remote 2>/dev/null || true)
 GIT_ROOT=$(git rev-parse --show-toplevel 2>/dev/null || true)
 GIT_TAGS=$(git describe --exact-match --tags HEAD 2>/dev/null || true)
 
-[[ "${CI}" != "true" ]] || {
-    [[ -z "${TRAVIS_BRANCH:-}" ]] || {
+if [ "${CI}" = "true" ]
+then
```

Comment:
for these small and benign changes, a separate PR that just flushes out one style for another is preferred to keep the history clean